Sources/RInAppMessaging/Utilities/BundleInfo.swift

```diff
@@ -32,6 +32,14 @@ internal class BundleInfo {
     class var inAppConfigurationURL: String? {
         return Bundle.main.infoDictionary?[Constants.Info.configurationURLKey] as? String
     }
+    
+    static var customFontNameText: String? {
+        Bundle.main.infoDictionary?[Constants.Info.customFontNameTextKey] as? String
```

Comment:
Isn't better to make it `var customFontNameText: String?` static so you can call `Bundle.main. customFontNameText`?
That can allow you to mock `customFontNameText` returned value.